dev/com.ibm.websphere.appserver.features/visibility/public/jca-2.0/io.openliberty.jca-2.0.feature

```diff
@@ -0,0 +1,18 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=io.openliberty.jca-2.0
+visibility=public
+singleton=true
+IBM-API-Package: com.ibm.ws.jca.service; type="internal"
```

Comment:
this line can be removed since com.ibm.ws.jca service is already exposed from the internal feature